src/bun.js/javascript.zig

```diff
@@ -3993,7 +3993,7 @@ pub const VirtualMachine = struct {
 
         if (show.system_code) {
             if (show.syscall) {
-                try writer.writeAll("  ");
+                try writer.writeAll("   ");
```

Comment:
is this supposed to be two spaces? and do we also need to print spaces before `syscall` in `if (show.syscall)`?

Reply:
it printed with one less in the situation i tried. maybe this whole logic should be redone to just compute the indent level of the : and then be able to compute the # of spaces per line.